### ISSUE TITLE
[kapacitor] HTTP Post URL not properly handled

### ISSUE THREAD

**Issue by jackzampolin** [influxdata/chronograf#1100]
<img width="693" alt="screenshot 2017-03-28 12 31 13" src="https://cloud.githubusercontent.com/assets/7452680/24423620/797cbfe6-13b2-11e7-8337-908e19409dc6.png">

When creating a task in kapacitor that needs to post it's result, chronograf does not include the user input in the URL field when creating the task in kapacitor.

The task will always have `.post('')`:

```js
var trigger = data
    |alert()
        .crit(lambda: "value" > crit)
        .stateChangesOnly()
        .message(message)
        .id(idVar)
        .idTag(idTag)
        .levelTag(levelTag)
        .messageField(messageField)
        .durationField(durationField)
        .post('')
```

**Comment by Tomcat-Engineering** [comment]
This bug applies to the SMTP alert config too.  The email addresses that you enter are ignored.  

The missing fields don't even make it into the HTTP PUT request, let alone the Kapacitor config.

Whenever you type in the relevant input elements you get this Javascript error:

```
Uncaught TypeError: Cannot read property 'alertProperty' of null
    at onChange (webpack:///./src/kapacitor/components/RuleMessageAlertConfig.js?:49:64)
```

Possibly related to [this commit](https://github.com/influxdata/chronograf/commit/9a3377d19ac7e9af265944cfdb031a7d37a5b3ef)?

**Comment by goller** [comment]
Hey @Tomcat-Engineering and @jackzampolin , thanks for writing in. We'll take a look! 

(also thanks for looking up that commit!)

**Comment by nhaugo** [comment]
Check if this fixes #1156
